contrib/src/main/java/com/datatorrent/contrib/enrich/FSLoader.java

```diff
@@ -128,6 +112,16 @@ public void setFileName(String fileName)
     return result;
   }
 
+  /**
+   * Extracts values from a record. Concrete implementations override this
+   * method to parse a record and convert it to Map of field names and values
```

Comment:
Can you add in the comment what happens when null is returned? I see both the implementations of this method returns null.